.github/workflows/build-push-docker.yaml

```diff
@@ -55,18 +55,22 @@ jobs:
           platforms: ${{ matrix.platform }}
           labels: ${{ steps.meta.outputs.labels }}
           outputs: type=image,name=${{ env.REGISTRY_IMAGE }},push-by-digest=true,name-canonical=true,push=true
+          build-args: |
+            APP_VERSION=${{ fromJSON(steps.meta.outputs.json).labels['org.opencontainers.image.version'] }}
```

Comment:
So for the image that you created from this branch the value for `APP_VERSION` is `feat-version` so it seems like it was derived from the branch name. Does 'it' automatically understand that the version should be `1.0.0-beta.6` once a new UniCore version is released?

Reply:
Yes, I inspected the meta information during "dirty" builds and proper release builds. If a git tag is there, it uses that one. I can also adjust the metadata generation and generate a `sha-<commit>` tag as well, if you prefer.

Reply:
Yeah actually I think it would be nice to have both!

Reply:
The tags have been updated and builds from untagged commits now look like so:

```
GET /version

{
  "version": "sha-9c10f4d"
}
```

Reply:
Great, nice!